index.html

```diff
@@ -11,8 +11,8 @@
       <div class="header-container-inner">
         <div class="logo-container"></div>
         <div class="navbar-container">
-          <a class="navbar-item" href="#/design">디자인</a>
-          <a class="navbar-item" href="#/dev">개발</a>
+          <a class="navbar-item" id="design" href="#/design">디자인</a>
+          <a class="navbar-item" id="dev" href="#/dev">개발</a>
```

Comment:
```suggestion
          <a class="navbar-item" id="design" href="/design">디자인</a>
          <a class="navbar-item" id="dev" href="/dev">개발</a>
```

`#` 표시를 빼면 더 좋을 것 같네요.

Reply:
해시라우팅으로 진행하면서, `#` 표시를 붙여서 진행되었었습니다.
History API로 변경하며 a태그 클릭시의 이벤트를 js상에서 동작하고있어, event.preventDefault(); 코드를 추가하였습니다.